chaco/plot_containers.py

```diff
@@ -129,51 +124,8 @@ class HPlotContainer(Container):
 
     draw_order = Instance(list, args=(DEFAULT_DRAWING_ORDER,))
 
-    # The dimension along which to stack components that are added to
-    # this container.
-    stack_dimension = Str("h", transient=True)
```

Comment:
we will want to set `transient=True` in `enable.stacked_container.StackedContainer`